domain/src/main/java/org/oppia/android/domain/topic/StoryProgressTestHelper.kt

```diff
@@ -446,4 +557,52 @@ class StoryProgressTestHelper @Inject constructor(
       timestamp
     )
   }
+
+  /**
+   * Marks one explorations in first test topic as completed played for a particular profile.
+   *
+   * @param profileId the profile we are setting recently played for
+   * @param timestampOlderThanOneWeek if the timestamp for the recently played story is more than a week ago
+   */
+  fun markChapterDoneFirstTestTopicStory0Exploration0(
+    profileId: ProfileId,
+    timestampOlderThanAWeek: Boolean
+  ) {
+    val timestamp = if (!timestampOlderThanAWeek) {
+      getCurrentTimestamp()
+    } else {
+      getOldTimestamp()
+    }
+    storyProgressController.recordCompletedChapter(
+      profileId,
+      TEST_TOPIC_ID_0,
+      TEST_STORY_ID_0,
+      TEST_EXPLORATION_ID_2,
+      timestamp
+    )
+  }
+
+  /**
+   * Marks one explorations in first test topic as completed for a particular profile.
+   *
+   * @param profileId the profile we are setting recently played for
+   * @param timestampOlderThanOneWeek if the timestamp for the recently played story is more than a week ago
```

Comment:
The parameter mention in the comment doesn't match with the parameter

```suggestion
   * @param timestampOlderThanAWeek if the timestamp for the recently played story is more than a week ago
```